WcaOnRails/lib/results_validators/scrambles_validator.rb

```diff
@@ -9,6 +9,8 @@ class ScramblesValidator < GenericValidator
     MISSING_SCRAMBLES_FOR_MULTI_ERROR = "[%{round_id}] While you may have multiple groups in 3x3x3 Multi-Blind, at least one of the groups must contain scrambles for all attempts."
     MULTIPLE_FMC_GROUPS_WARNING = "[%{round_id}] There are multiple groups of FMC used. If one group of FMC was used, please use the Scrambles Matcher to uncheck the unused "\
       "scrambles. Otherwise, please include a comment to WRT explaining why multiple groups of FMC were used."
+    WRONG_NUMBER_OF_SCRAMBLE_SETS_ERROR = "[%{round_id}] Has a different number of scrambles than specified on the Edit Events tab. "\
+      "Please adjust the number of scramble sets in the Edit Events tab to the number of attempts that were used."
```

Comment:
I would suggest: 

"[%{round_id}] **This round has** a different number of **scramble sets** than specified on the **Manage** Events tab. Please adjust the number of scramble sets in the **Manage** Events tab to the number of **sets** that were used."